### PR TITLE
Update for issue #6

### PR DESCRIPTION
**Pull request by PokeyCA**
This should allow rounds to go beyond 9 and update minutes.

### CODE REVIEW

**Comment by PokeyCA** [comment]
I hope I did this correctlyish, as this is my first time working on github to assist.

File: scripts/longtermeffects.lua [hunk 1]
```diff
@@ -61,14 +61,12 @@ function nextRound_new(nRounds, bTimeChanged)
...
-		-- bmos resetting rounds and advancing time
-		if nCurrent >= 10 and not bTimeChanged then
-			local nMinutes = math.floor(nCurrent / 10)
-			nCurrent = nCurrent - (nMinutes * 10)
-			CalendarManager.adjustMinutes(nMinutes)
+		-- bmos advancing time
+		if (nCurrent % 10) = 9 and not bTimeChanged then
+			CalendarManager.adjustMinutes(1)
 			CalendarManager.outputTime()
 		end
```
Comment (Corun-Kail) [inline scripts/longtermeffects.lua]: If we are no longer having side effect of nCurrent changing I would suggest to extract both this fragment and next occurrence into a function.

Comment (bmos) [reply scripts/longtermeffects.lua:68]: I think it needs to be == here rather than =.

**Comment by bmos** [comment]
This is interesting. Never knew there were modulus or exponent operators in lua.

The only thing lost here is the ability to advance time via the combat tracker (barely a loss at all IMO).
I think it's probably best to make the rounds field read-only, but that could be troublesome for people.

**Comment by PokeyCA** [comment]
Well, the change lets the rounds continue counting up and adding a minute every time 10 rounds go by.

**Comment by bmos** [comment]
Understood. I'm excited for that functionality but am worried about edge cases (which is why I didn't construct it in this manner originally). Looking forward to playing with it this afternoon.

**Comment by bmos** [comment]
the issue with this is that it goes to 19 and then reverts to 9 rather than 0 or 20

**Comment by PokeyCA** [comment]
It looks like some of the changes reverted when it got pulled back from Clock Adjuster.

**Comment by bmos** [comment]
awesome! thanks @PokeyCA !

**Pull request merged**